### ISSUE TITLE
Mark tweets as favorite

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#57]
```
I would like to be able to mark a tweet as favorite, cp. 
http://twitter.com/#favorites

A star is a well established symbol for this. So I can see a star in the top-
right corner or in the bottom-right corner of a tweet doing just fine.

```

Original issue reported on code.google.com by `alex.bepple` on 31 Jan 2010 at 9:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 31 Jan 2010 at 10:36
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Issue 159 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 29 Mar 2010 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
Closed by 059a4cb5c5 revision.
```

Original comment by `tro...@gmail.com` on 29 Mar 2010 at 1:54
- Changed state: **Fixed**
